app/models/user.js

```diff
@@ -0,0 +1,30 @@
+module.exports = (sequelize, DataTypes) => {
+  const user = sequelize.define('user', {
```

Comment:
we recommend calling it User, something like `const User = sequelize.define('User', {`

Reply:
model name fixed